submissions/crealime/dom-api/css/reset.css

```diff
@@ -0,0 +1,4 @@
+html, body, div, span, applet, object, iframe, blockquote, pre, a, abbr, acronym, address, big, cite, code, del, dfn, em, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby, section, summary, time, mark, audio, video {
+  margin: 0;
+  padding: 0
+}
```

Comment:
Read about [normalize](https://necolas.github.io/normalize.css/) or [reset](https://meyerweb.com/eric/tools/css/reset/) css

Reply:
I added standard reset. Although I modified it a little because it breaks my work with rem.

Reply:
@crealime heh, as I understand, you use just part of reset. But, ok it's not a problem in this case.